docs/docs/installation/pypi.mdx

```diff
@@ -100,24 +98,18 @@ We highly recommend installing Superset inside of a virtual environment. Python
 pip install virtualenv
 ```
 
-You can create and activate a virtual environment using:
-
+First create a directory for Superset:
 ```
-# virtualenv is shipped in Python 3.6+ as venv instead of pyvenv.
-# See https://docs.python.org/3.6/library/venv.html
-python3 -m venv venv
-. venv/bin/activate
+mkdir superset
```

Comment:
Hmmm. I'm a tad confused. Shouldn't the virtualenv be called `venv`—assuming it's inside the root `superset` folder?

Reply:
Right now the docs don't say to make a folder called `superset`, that's something I'm adding here.  I believe what they say currently makes a virtualenv called `venv` in the root directory, which is not descriptive and potentially a problem if another tutorial says to do the same.

Now maybe `mkdir superset` is unnecessary if `python3 -m venv superset` below would create the directory.

I may be misunderstanding though, I'm new to this aspect.